src/helpers/environment.sh

```diff
@@ -16,6 +17,17 @@ function ensure_git_repository {
 }
 
 
+function gittown_dev_version {
+  local base_dir=$( dirname "${BASH_SOURCE[0]}" )
+  local git_hash=$(git -C "$base_dir" rev-parse --short HEAD)
+  local git_date=$(git -C "$base_dir" --no-pager show -s --format=%cD HEAD | cut -d ' ' -f2-4)
```

Comment:
Dont calculate these unless you need them (move then to the inside of the if statement.